python/Makefile.am

```diff
@@ -232,8 +232,9 @@ adjoint_PYTHON  = $(srcdir)/adjoint/__init__.py              \
 PY_PKG_FILES = $(INIT_PY) $(HL_IFACE) .libs/_meep.so
 
 meep: _meep.la $(MPB_LA) __init__.py $(HL_IFACE)
-	mkdir -p meep
+	mkdir -p meep/adjoint
 	cp $(PY_PKG_FILES) meep
+	cp $(adjoint_PYTHON) meep/adjoint
```

Comment:
What is going on with this change?

Reply:
Before this change, only the `$PY_PKG_FILES` were copied to the `meep` folder.
In a second step, the MPB interface was added below.
However, the `adjoint` subpackage was never copied to `meep`.

Do you have any insight why that is the case?

Reply:
Our Python Makefile rules seem kind of hacky here; it would be better to rewrite them to cooperate with automake if possible.